Stream-Filter/README.md

```diff
@@ -0,0 +1,12 @@
+OpenTok.js Stream Filter Sample
+=======================
+
+As of v2.13 of opentok.js you can pass a custom videoSource and custom audioSource to the Publisher. This sample shows how to use this API to apply custom filters to a Publisher. It is very similar to the [Basic Video Chat](../Basic%20Video%20Chat/) example but it adds a [publish.js](./js/publish.js) and [filters.js](./js/filters.js) files which handle publishing a custom videoSource and audioSource.
```

Comment:
I think a little more explanation would be useful. I would say something like the following:

You can set a custom audio source and video source for a publisher's stream when you call `OT.initPublisher()`. The custom audio and video source are based on a MediaStream track. In this example, we use the `OT.getUserMedia()` (from OpenTok.js) to get a reference to a MediaStream object that uses the microphone and camera as the audio and video source. The sample attaches the video MediaStreamTrack (from the MediaStream object) to an HTML canvas, manipulates the canvas's image (for example, applying a grayscale filter). It then obtains the resulting MediaStreamTrack from the canvas, using the `Canvas.captureStream()` method to get a MediaStream object, and it calls `getVideoTracks()[0]` on that object to get the video MediaStreamTrack object. Finally, it uses that MediaStreamTrack object when calling `OT.initPublisher()`.